.golangci.yml

```diff
@@ -63,11 +63,11 @@ linters:
     - unconvert
     - unparam
     - varcheck
-    - whitespace
+#    - whitespace
 #    - wsl
 
 issues:
-  fix: true
+  fix: false
```

Comment:
I think we should leave `fix` on, and regardless it should be changed in its own PR if it's to be changed. Same for rules above.

Reply:
I've reverted the linter changes 775ce28. Will do a separate PR for this.

The preliminary arguments to disable both are:
1. Nothing should mess with the code because it is unpredictable what's going to be changed and where. The sole linter task is to issue warnings/errors.
2. Whitespace rules are especially annoying. For some functions I use extra new line after each statement to greatly improve readability. This is not only beneficial for myself, but other team members do have easier time to read/review the critical code paths.
3. The eventual source of truth for Go code formatting is gofmt, and it has not issues with extra new lines where I use it.